app/assets/javascripts/govuk-admin-template/modules/track_button_press.js

```diff
@@ -0,0 +1,16 @@
+(function(Modules) {
+  "use strict";
+
+  Modules.TrackButtonPress = function() {
+    var that = this;
+    that.start = function(element) {
+      element.find(".js-track").on("click", function() {
+        var action = $(element).data("track-action") || "button-pressed",
```

Comment:
I believe `element.data` should work here as the `element` var is already wrapped with the jQuery helpers.